workflows/pi_correspondence/email_templates/A_initial_email_template.Rmd

```diff
@@ -1,86 +1,99 @@
 
 File contents and relationships among files are clear
 
-> Could you provide a short description of the files submitted? Information about how each file was generated (what software, source files, etc.) allows other scientists to reproduce your work.
+> Could you provide a short description of the files submitted? Information about how each file was generated (what software, source files, etc.) will help us create more robust metadata for long term use.
+
+We try not to prescribe a way the researchers must format their data as long as reasonable. However, in extreme cases (for example Excel spreadsheets with data and charts all in one sheet) we will want to kindly ask them to reformat.
+
+> We would like to suggest a couple of modifications to the structure of your data. This will others to re-use it most effectively. [DESCRIBE WHAT MAY NEED TO BE CHANGED IN THE DATA SET]. Our data submission guidelines page (https://arcticdata.io/submit/) outlines what are best practices for data submissions to the Arctic Data Center. Let us know if you have any questions or if we can be of any help.
+
+#### Attributes
+
+> I noticed that your data package has incomplete attribute/variable information. [SPECIFY WHAT YOU WANT ADDED] You can do so by navigating to your dataset [INSERT URL HERE], clicking the green "Edit" button, clicking the "Describe" button located to the right of each of your files at the top of the submission page, and in the window that pops up click over to the "Attributes" tab. Please see the guidance below for what attribute/variable information should contain.
+
+Checklist for the datateam in reviewing attributes (NetCDF, CSV, shapefiles, or any other tabular datasets):
+> - A name (often the column or row header in the file).
+> - A complete definition.
+> - Any missing value codes along with explanations for those codes.
+> - For all numeric data, unit information is needed.
+> - For all date-time data, a date-time format is needed (e.g. "DD-MM-YYYY").
+> - For text data, full descriptions for all patterns or code/definition pairs are needed if the text is constrained to a list of patterns or codes.
+
+> Could you describe ____? 
+> Please define “XYZ”, including the unit of measure. 
+> What are the units of measurement for the columns labeled “ABC” and “XYZ”?
 
 Missing value codes
 
-> What do the missing values in your measurements represent? (instrument failure, site not found, etc.)
+> What do the missing values in your measurements represent? Something such as a short description (instrument failure, site not found, etc.) will suffice.
 
-> We noticed that the data files contain blank cells. What do these represent? 
+> We noticed that the data files contain [blank cells - replace with missing values found]. What do these represent? 
```

Comment:
I'm not totally sure what the revision means here. Is replace with missing values found an instruction for the email sender?